hbw/app/javascript/packs/hbw/components/shared/utils/date_format_converter.js

```diff
@@ -0,0 +1,5 @@
+// date-fns uses 'd' and 'y' for month days and calendar years
+// as opposed to moment.js that uses capital 'D' and 'Y'
+const convertToDateFNSDateFormat = dateFormat => dateFormat.replaceAll('Y', 'y').replaceAll('D', 'd');
+
+export default convertToDateFNSDateFormat;
```

Comment:
I would expect this file to be named `to_fns_date_format` rather than this clumsy noun-based name